gradle.properties

```diff
@@ -26,26 +26,32 @@ jacocoVersion=0.8.10
 ballerinaToOpenApiVersion=2.1.0
 swaggerCoreVersion=2.2.22
 
-stdlibIoVersion=1.6.1
-stdlibTimeVersion=2.4.0
-stdlibUrlVersion=2.4.0
+githubSpotbugsVersion=6.0.18
+githubJohnrengelmanShadowVersion=8.1.1
+underCouchDownloadVersion=5.4.0
+researchgateReleaseVersion=2.8.0
 
-stdlibConstraintVersion=1.5.0
-stdlibOsVersion=1.8.0
-stdlibTaskVersion=2.5.0
-stdlibLogVersion=2.10.0
-stdlibCryptoVersion=2.7.2
+stdlibIoVersion=1.6.2-20241112-233100-995cf5f
+stdlibTimeVersion=2.6.0-20241113-073800-201b904
+stdlibUrlVersion=2.4.1-20241113-073900-335ff51
 
-stdlibFileVersion=1.10.0
-stdlibMimeVersion=2.10.0
-stdlibCacheVersion=3.8.0
+stdlibConstraintVersion=1.6.0-20241113-090900-d276ad5
+stdlibOsVersion=1.8.1-20241113-122000-cca973b
+stdlibTaskVersion=2.5.1-20241113-123500-f905281
+stdlibLogVersion=2.10.1-20241113-120000-4577868
+stdlibCryptoVersion=2.7.3-20241113-081400-d015a39
 
-stdlibAuthVersion=2.12.0
-stdlibDataJsonDataVersion = 0.3.0-20241105-101100-661d11f
-stdlibJwtVersion=2.13.0
-stdlibOAuth2Version=2.12.0
+stdlibFileVersion=1.10.1-20241113-151700-e1a2e38
+stdlibMimeVersion=2.10.2-20241113-154200-d953747
```

Comment:
Shall we update the latest timestamped version? There is a change in the API which used to write streams: https://github.com/ballerina-platform/module-ballerina-mime/pull/546/commits/50d8dca19127899acae2065297f61d34d866d89e